classScreen.js

```diff
@@ -0,0 +1,34 @@
+(function(global) {
+    "use strict";
+
+    function Screen(element) {
```

Comment:
Since we learned how to put a class in a dedicated script, our next goal is to support problem at a larger scale. Imagine that you are working on a small project with 1000 classes -- it is natural to have name conflicts. For example, in the same project you have a dedicated screen for calculator (like here), but also several more screens for: time, latests news etc. In order to avoid name collisions in programming world, we use namespaces which gather up classes with similar purpose. If class was a file a namespace would be folder(s). Check some online tutorials just to understand the concept (from Java language).

Since JavaScript, does not have support for namespaces, we have to create it on our own. We need to make a mechanism which will enable is to use namespace-like mechanism. Basically it needs to have two **global** functions: *require* & *provide*. With provide, you'll export something, with require you'll import something.

An example:
```
(function() {
 var Screen = function() { ... };
 provide('calculator.Screen', Screen); // export
});

(function() {
 var Screen = function() { ... };
 provide('clock.Screen', Screen); // export
});


(function() {
 var CalculatorScreen = require('calculator.Screen'); // import
 var ClockScreen = require('clock.Screen'); // import
 // do something
});
```

Your task is to create a module with two global functions *require* and *provide*, using which we can simulate namespaces like in the example above. Internally that module will use an object to map a string to a class.